constants/ChangedChatErrors.json

```
@@ -103,6 +103,12 @@
         "TPS calculation got an error"
       ],
       "fixed_in": "1.12.0"
+    },
+    {
+      "message_starts_with": [
+        "Could not read category for item"
+      ],
+      "fixed_in": "2.99.0"
```

```suggestion
      "fixed_in": "2.1.0"
```
Would be a lot more sensible